src/Microsoft.ML.PipelineInference/AutoInference.cs

```diff
@@ -370,19 +371,21 @@ public void UpdateTerminator(ITerminator terminator)
                 TransformInference.SuggestedTransform[] existingTransforms = null)
             {
                 // Infer transforms using experts
-                var levelTransforms = TransformInference.InferTransforms(_env, data, args);
+                var levelTransforms = TransformInference.InferTransforms(_env, data, args, this._columnPurpose);
```

Comment:
>this [](start = 91, length = 4)

we tend to omit usage of "this" word.

Reply:
Thanks Ivan. I will take care of this in the next iteration

Reply:
Fixed

---
In reply to: [199589164](https://github.com/dotnet/machinelearning/pull/461#discussion_r199589164) [](ancestors = 199589164)